src/pivot-table/data/headers-data.ts

```diff
@@ -6,15 +7,19 @@ const createHeadersData = (
   visibleTopDimensionInfo: VisibleDimensionInfo[],
   visibleLeftDimensionInfo: VisibleDimensionInfo[],
 ): HeadersData => {
-  // rowCount cannot be 0, as it couse issue when there is no top data but there is left data
   const data = extractHeaders(layoutService, visibleTopDimensionInfo, visibleLeftDimensionInfo);
-
+  const size = {
+    x: data[0]?.length || 0, // number of columns
```

Comment:
Nitpicking but this should be the nullish coalescing operator
```suggestion
    x: data[0]?.length ?? 0, // number of columns
```

Reply:
done